requirements.txt

```diff
@@ -3,7 +3,7 @@ ovos-utils<0.2.0,>=0.0.27
 ovos-config~=0.0.5
 youtube-search~=2.1
 pytube~=12.1
-websockets~=10.4
+websockets>=10.4
```

Comment:
Is this intentionally allowing an unpinned maximum version? Consider adding a `<12.0` if this is to allow `11.x` in case there are breaking changes in a future version

Reply:
After reading the commit note, maybe `<13.0` since this is there to allow `12.X`

Reply:
Still concerned about this unpinned max version in case the websockets package introduces a breaking change

Reply:
Did I not push an update? I remember committing one...

Reply:
consider moving to websocket-client>=0.54.0 instead, as it is already a dependency of ovos-bus-client and always available, always good to keep dependencies to a minimum and both of these do the same  thing

aditionally, i dont remember what package it was, but websockets~=10.4  was conflicting with something 3rd party dependency too